### PR TITLE
feat(classification): update security controls watermarking copies

### PR DESCRIPTION
**Pull request by bxie-box**
**Short Summary**:
Short Summary

**Modal**:
Modal

### CODE REVIEW

File: src/features/classification/flowTypes.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ type Controls = {
 type ControlsFormat = $Values<typeof SECURITY_CONTROLS_FORMAT>;
 
 type MessageItem = {
-    message: MessageDescriptor,
+    message: React.Node,
```
Comment (jpfranco) [inline src/features/classification/flowTypes.js:62]: I think this should be `MessageDesciptor | React.Node`

File: src/features/classification/security-controls/SecurityControlsItem.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ const ICON_SIZE = 13;
 
 const SecurityControlsItem = ({ message, tooltipMessage }: Props) => (
     <li className="bdl-SecurityControlsItem">
-        <FormattedMessage {...message} />
+        {React.isValidElement(message) ? message : <FormattedMessage {...message} />}
```
Comment (jpfranco) [inline src/features/classification/security-controls/SecurityControlsItem.js:18]: Can we add a unit test for when `isValidElement` returns true?

**Review by jpfranco — COMMENTED** [review]
Just some minor comments, but I think they might be worth fixing.

**Comment by bxie-box** [comment]
Adding multiple restrictions for ref
Multiple Restrictions

**Comment by bxie-box** [comment]
Have been trying to resolve the flow error, but going to look into it more tomorrow.

**Comment by bxie-box** [comment]
Deferring to the use of `$FlowFixMe` per offline discussion (had spent a bit on addressing this /  few approaches with type casting etc)

**Review by patrickpaul — APPROVED** [review]

**Pull request merged**